central/imagev2/datastore/datastore_impl_test.go

```diff
@@ -0,0 +1,646 @@
+//go:build sql_integration
```

Comment:
Probably worth looking to see if all these tests provide separate value or if we can combine or remove any.

Reply:
Do you mean merging datastore_impl_test, fixable_test and datastore_sac_test ?

Reply:
not necessarily.  Mostly I was thinking, looking through them and removing things that are duplicative.